internal/config/app_domain.go

```diff
@@ -1,6 +1,6 @@
 package config
 
 type AppDomainConfig struct {
-	Domain string `json:"domain" pageship:"required,max=200,hostname_rfc1123,lowercase"`
+	Domain string `json:"domain" pageship:"required,max=200,hostname_port|hostname_rfc1123,lowercase"`
```

Comment:
I don't think we need port? Domain verification/association should not care about the port.